test/index.html

```diff
@@ -11,40 +11,45 @@
     <h1>Checka11y.css - test.html</h1>
     <p>The intention of this file is to test out the new a11y checks you are adding, so this web page should try to not comply with a11y as much as possible.</p>
     <hr />
+    <section>
+      <h2>HTML: html tag should have a lang attribute</h2>
+      <p>See top of page.</p>
+    </section>
     <section>
       <h2>Images: All images require an alt attribute, whether it has a value or is null</h2>
       <img src="./assets/a11y.png" />
     </section>
     <section>
       <h2>iframe: iframe has title attribute</h2>
-      <iframe src="https://jackdomleo.dev" height="500" width="500"></iframe>
+      <iframe src="https://checka11y.jackdomleo.dev" height="500" width="500"></iframe>
     </section>
     <section>
       <h2>Lists: All children of &lt;ol&gt; must only be &lt;li&gt;</h2>
       <ol>
-        <div>Hello, I'm a div inside a &lt;ol&gt;</div>
+        <li>Hello, I'm a &lt;li&gt; inside a &lt;ol&gt;</li>
+        <div>Hello, I'm a &lt;div&gt; inside a &lt;ol&gt;</div>
+        <li>Hello, I'm a &lt;li&gt; inside a &lt;ol&gt;</li>
       </ol>
     </section>
     <section>
       <h2>Lists: All children of &lt;ul&gt; must only be &lt;li&gt;</h2>
-      <ol>
-        <div>Hello, I'm a div inside a &lt;ul&gt;</div>
-      </ol>
+      <ul>
+        <li>Hello, I'm a &lt;li&gt; inside a &lt;ul&gt;</li>
+        <div>Hello, I'm a &lt;div&gt; inside a &lt;ul&gt;</div>
+        <li>Hello, I'm a &lt;li&gt; inside a &lt;ul&gt;</li>
+      </ul>
     </section>
     <section>
       <h2>Navigation: Only a single navigation element should exist</h2>
       <nav>This is the first navigation</nav>
       <nav>This is the second navigation</nav>
     </section>
-    <section>
-      <h2>HTML: html tag should have a lang attribute</h2>
-    </section>
     <section>
       <h2>Description list: Direct children of a &lt;dl&gt; can only be &lt;dt&gt; and &lt;dd&gt;</h2>
       <dl>
-        <dt>Hello, I'm dt element inside dl element</dt>
-        <dd>Hello, I'm dd element inside dl element</dd>
-        <p>Hello, I'm p(paragraph) element inside dl element. </p>
```

Comment:
I'm a paragraph element, remove the brackets and the leading p

Reply:
I think I removed this section and edited it to something more appropriate. This was something I accidentally missed from another PR.